addOns/scripts/CHANGELOG.md

```diff
@@ -4,7 +4,11 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
+### Added
+- Support for loadDir action, which loads all of the scripts under the specified directory..
 
+### Changed
+- File parameter to `path`, `file` will still work.
```

Comment:
I'd still suggest `source`, we can support later other ways than the file system.

Reply:
Sorry, missed that comment!